src/components/Project/FeedbackModule.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import Section from '../General/Section';
+import LongCard from '../General/LongCard';
+import styled from 'styled-components';
+import {breakpoint} from '@aragon/ui';
+const medium = css => breakpoint('medium', css);
+const large = css => breakpoint('large', css);
+import speech from './assets/speech.svg';
+
+const FeedbackModule = () => (
+  <StyledSection>
+    <Container>
+      <Feedback>
+        <ImageContainer>
+          <img src={speech} alt="feedback" />
+        </ImageContainer>
+        <div>
+          <h1>We value the feedback & ideas from the community</h1>
+          <span>
+            <p>
+              Please let us know if there something important for you that is
+              not included in this roadmap and we’ll take it into consideration
+              when planning our quaterly efforts.
+            </p>
+            <p>
+              <a
+                className="action-button"
+                href="https://forum.aragon.org"
+                target="_blank">
+                <span>
+                  Join the Aragon forum{' '}
+                  <img src={require(`../General/assets/arrow.svg`)} alt=">" />
+                </span>
+              </a>
+            </p>
+          </span>
+        </div>
+      </Feedback>
+    </Container>
+  </StyledSection>
+);
+
+const StyledSection = styled(Section)`
+  padding: 100px 15px;
+`;
+const Container = styled.div`
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: auto;
+  grid-column-gap: 30px;
+  grid-row-gap: 30px;
+  ${medium(
+    'grid-template-columns: calc(50% - 15px) calc(50% - 15px); grid-template-rows: auto'
+  )};
+`;
+
+const Feedback = styled.div`
+  border-radius: 12px;
+  box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.07);
+  background-color: #f9fafc;
+  padding: 30px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  height: 100%;
+  grid-column-start: 1;
+  grid-column-end: 2;
+  transition: all 0.25s ease-in-out;
+  ${medium('padding: 30px 15px; grid-column-end: 3; flex-direction: row;')};
```

Comment:
I really like the shorthand `medium` function you created, but if you use the original one you can get an indentation that resembles the rest of your `css`:

```JavaScript
const Feedback = styled.div`
  ${breakpoint(
  'medium',
  `
    padding: 30px 15px;
    grid-column-end: 3;
    flex-direction: row;
  `)}
`
```